src/cockpit/389-console/src/lib/server/auditLog.jsx

```diff
@@ -447,6 +556,45 @@ export class ServerAuditLog extends React.Component {
                                 />
                             </FormGroup>
                         </Form>
+                        <Form className="ds-margin-top-lg ds-margin-left" isHorizontal autoComplete="off">
+                            <FormGroup
+                                label="Display Attributes"
+                                fieldId="nsslapd-auditlog-display-attrs"
+                                title="Display attributes from the entry in the audit log (nsslapd-auditlog-display-attrs)."
+                            >
+                                <div className={this.state.displayAllAttrs ? "ds-hidden" : "ds-margin-bottom"}>
+                                    <Select
+                                        variant={SelectVariant.typeaheadMulti}
```

Comment:
Hmm, it doesn't work as multi-valued for me... If I add two attributes and click `Save Log Setting` - it says `Success!` but after refreshing shows only one attribute.